src/app/search-field/search-field.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, Output, EventEmitter } from '@angular/core';
+
+@Component({
+  selector: 'app-search-field',
+  templateUrl: 'search-field.component.html'
+})
+export class SearchFieldComponent {
+
+  input: string;
+  submitted = false;
+
+  @Output()
+  nameSubmitted: EventEmitter<string> = new EventEmitter<string>();
+
+  onSubmit() {
+    this.submitted = true;
+    this.nameSubmitted.emit(this.input);
+    console.log('emitted nameSubitted');
```

Comment:
obsolet?

Reply:
Nö, ist jetzt 'n generisches Suchfeld, dass man weiterverwenden kann und in der Kartensuche auch wird.